### PR TITLE
Support passing extra connection options to the Mongo driver

### LINKED ISSUE #10526: Add support for passing connection options to Mongo
### Description:
It's not currently possible to provide [connection settings](http://mongodb.github.io/node-mongodb-native/2.2/reference/connecting/connection-settings/) to the Mongo driver with Rocket Chat. This is required to provide a PEM file and CA cert file if Mongo is configured to only accept connections from clients with valid certs. Simply adding `ssl=true` to the `MONGO_URI` isn't sufficient because there's no way to pass the `sslCA` and `sslCert` parameters in the URI.

Meteor [supports passing extra options](https://github.com/meteor/meteor/blob/devel/packages/mongo/connection_options.js) to the underlying mongo driver. Rocket chat should provide a way for connection options to be configured to allow this usecase.

### Server Setup Information:

* Version of Rocket.Chat Server: 0.63.3
* mongoDB Version: 3.6

### Steps to Reproduce:

1. Create a mongo replica set secured with TLS.
2. Try to configure Rocket chat to connect to mongo with a verified SSL certificate.

### Expected behavior:
Rocket chat should provide a mechanism to configure the mongo driver to pass the required parameters to connect via TLS with verified certificates.

### Actual behavior:
It's not possible to configure the mongo driver in Rocket chat so it can't connect.

### PR DESCRIPTION
**Pull request by saplla**
Fixes #10526

### CODE REVIEW

**Comment by saplla** [comment]
I can't see anywhere obvious to document this. Any pointers?

File: server/startup/serverRunning.js [hunk 1]
```diff
@@ -13,6 +13,12 @@ Meteor.startup(function() {
 		}
 	}
 
+	var mongoOptionStr = process.env.MONGO_OPTIONS;
+	if (typeof(mongoOptionStr) !== 'undefined') {
```
Comment (graywolf336) [inline server/startup/serverRunning.js]: Let's go with `typeof mongoOptionStr` to stay consistent with the rest of our codebase.

**Comment by saplla** [comment]
@sampaiodiego Done.
@graywolf336 Done.

**Comment by saplla** [comment]
Documentation PR at https://github.com/RocketChat/docs/pull/707

**Comment by saplla** [comment]
@graywolf336 / @sampaiodiego any chance you could review this today please?

**Pull request merged**